### PR TITLE
Added AntD mobile UI

### PR DESCRIPTION
**Pull request by shivanshs9**
- Homepage for mobile done
- Added webpack resolve alias
- Added theme scss
- Added ImageButton and FeedbackButton

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -36,9 +37,9 @@
     "typeface-roboto": "^0.0.75"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
+    "start": "react-app-rewired start",
```
Comment (robinv85) [inline client/package.json:40]: Why did you need these script overrides?

File: client/package.json [hunk 2]
```diff
@@ -57,7 +58,8 @@
     ]
   },
   "devDependencies": {
-    "node-sass": "^4.13.1"
+    "node-sass": "^4.13.1",
```
Comment (robinv85) [inline client/package.json:61]: So we should remove node-sass and scss and Grommet and Bootstrap and only keep Ant and Styled Components, but we can do that in the next iteration and merge this first

File: client/src/index.scss [hunk 3]
```diff
@@ -1,14 +1,10 @@
...
-@import "~bootstrap/scss/bootstrap.scss";
+// Import Theme Style
+@import "./styles/theme.scss";
 
 body {
   margin: 0;
-  font-family: "Poppins", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans",
-    "Droid Sans", "Helvetica Neue", sans-serif;
+  background: $body-bg;
+  font-family: $font-family;
```
Comment (robinv85) [inline client/src/index.scss:7]: yes can probably remove these things and move them to Styled Components theme

File: client/package.json (hunk 1 above)
Comment (shivanshs9) [reply client/package.json:40]: To extend webpack config.. I didn't want to replace the config provided by create-react-app.
And I needed to configure webpack for aliases

File: client/src/index.scss (hunk 3 above)
Comment (shivanshs9) [reply client/src/index.scss:7]: You mean using ThemeProvider?

Comment (robinv85) [reply client/src/index.scss:7]: yes I guess replace what we had before to use https://styled-components.com/docs/advanced#theming
